packages/template-retail-react-app/app/components/item-variant/item-price.jsx

```diff
@@ -105,6 +107,30 @@ const ItemPrice = ({currency, align = 'right', baseDirection = 'column', ...prop
                     <PricePerItem currency={currency} basePrice={basePrice} basket={basket} />
                 </HideOnMobile>
             )}
+
+            {/*This area is purely for screen reader*/}
+            <VisuallyHidden aria-live="polite" aria-atomic={true}>
```

Comment:
Why we create a seperate section instead of using `aria-live` here? Since cart page can have various items, we need to give more context (e.g product name) when the prices change here rather than just announcing the newly updated prices